content/guides/comparison.md

```diff
@@ -10,6 +10,8 @@ contributors:
 
 webpack is not the only module bundler out there. If you are choosing between using webpack or any of the bundlers below, here is a feature-by-feature comparison on how webpack fares against the current competition.
 
+<div class="scroll-table">
```

Comment:
Ok let me know once you've pushed the changes, I'll test again and we should be good to go.